targets/bmv2/CMakeLists.txt

```diff
@@ -4,6 +4,17 @@ if(ENABLE_TESTING)
   include(test/P4Tests.cmake)
 endif()
 
+find_program(RTSMITH_DRIVER p4rtsmith)
```

Comment:
We already set this binary with a variable, you just need to use that variable. Do not use find_program